### PR TITLE
feat: add some build scripts

### PR DESCRIPTION
**Pull request by j8xixo12**
As title, I added bzip2, xz, cmake, openssl, python, zlib build scripts.

### CODE REVIEW

**Review by tai271828 — APPROVED** [review]
Looks good to me. In my opinion, this is the very first commit for build scripts, and each of them are modularized very much. I incline to land the code if there is no critical and explicit pitfall, and apply rolling update later to make it perfect.

File: scripts/build.d/openssl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Copyright (C) 2011 Yung-Yu Chen <yyc@solvcon.net>.
```
Comment (yungyuc) [inline scripts/build.d/openssl:3]: You don't need this line of copyright.

File: scripts/build.d/python [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (C) 2011 Yung-Yu Chen <yyc@solvcon.net>.
```
Comment (yungyuc) [inline scripts/build.d/python:3]: Ditto.

File: scripts/build.d/zlib [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/env bash
+#
+# Copyright (C) 2011 Yung-Yu Chen <yyc@solvcon.net>.
```
Comment (yungyuc) [inline scripts/build.d/zlib:3]: Ditto

**Review by yungyuc — APPROVED** [review]
LGTM.  The copyright lines can be dropped with this or in a future PR.

**Comment by yungyuc** [comment]
@j8xixo12 Please merge as you see fit.

**Pull request merged**